### PR TITLE
[terra-status-view] Updates message content to use semantic html element.

### LINKED ISSUE #2697: [terra-status-view] Accessibility issues with <div> usage
# Bug Report

The status view title and message should use appropriate semantic HTML elements instead of divs to meet accessibility requirements. At the very least, the divs should be replaced with paragraph elements. Ideally, the status view title would be able to map to various header elements.

## @ Mentions
@scottwilmarth @neilpfeiffer

### PR DESCRIPTION
**Pull request by lokesh-0813**
Fixes #2697

### CODE REVIEW

**Comment by lokesh-0813** [comment]
There were some difference in few of the wdio screenshots and hence were re-generated.

File: packages/terra-status-view/src/StatusView.jsx [hunk 1]
```diff
@@ -118,9 +118,9 @@ const StatusView = ({
   let messageSection;
   if (message) {
     messageSection = (
-      <div className={cx('message')}>
+      <p className={cx('message')}>
```
Comment (StephenEsser) [inline packages/terra-status-view/src/StatusView.jsx]: Regarding the screenshot changes, we should be able to style the paragraph tag to realign with the styles the div was providing previously.

Comment (lokesh-0813) [reply packages/terra-status-view/src/StatusView.jsx]: Styles adjusted [here](544daee1958f769bd61b6c9d1d10882921de08ea).

File: packages/terra-status-view/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* `<div>` tag replaced with `<p>` in message content.
```
Comment (yuderekyu) [inline packages/terra-status-view/CHANGELOG.md]: can we specify that this change is for a11y?

**Comment by jeremyfuksa** [comment]
In the issue discussion, @neilpfeiffer suggested both the error title and the message be rendered as `<p>` elements. I'm only seeing that the message was changed to `<p>`.

File: packages/terra-status-view/CHANGELOG.md (hunk 2 above)
Comment (lokesh-0813) [reply packages/terra-status-view/CHANGELOG.md]: Updated here 8f59698af7fd91b2b9c32979400bc90749734ce6

**Comment by lokesh-0813** [comment]
> In the issue discussion, @neilpfeiffer noted that both the error title and the message be rendered as `<p>` elements. I'm only seeing that the message was changed to `<p>`.
> 
> [Title](https://github.com/cerner/terra-core/blob/444e672dd2d5822b1d6c675b04e716ddb27e0431/packages/terra-status-view/src/StatusView.jsx#L112) needs to be changed as well, making sure that no global paragraph styling bleeds in.

@jeremyfuksa I misunderstood the comment.
Here is a change to title content 8f59698af7fd91b2b9c32979400bc90749734ce6

**Pull request merged**